src/main/java/org/schemaspy/DbDriverLoader.java

```diff
@@ -148,13 +157,34 @@ protected Driver getDriver(String driverClass, String driverPath) throws Malform
         return driver;
     }
 
+    private void loadAdditionalJarsForDriver(String driverPath, Set<URL> classpath) throws MalformedURLException {
+        File driverFolder = new File(Paths.get(driverPath).getParent().toString());
+
+        if (driverFolder != null) {
+            File[] files = driverFolder.listFiles(
+                    (dir, name) -> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement and then remove useless return keyword [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1602)